Library Catalog (Aleph).js

```diff
@@ -302,76 +302,6 @@ function scrape(newDoc, marc, url) {
 
 /** BEGIN TEST CASES **/
 var testCases = [
-	{
-		"type": "web",
-		"url": "http://naude.bibliotheque-mazarine.fr/F/GFVQXDP767A179G9GAIRLC17TQHRXP67DJCKQMH28VKPBPKCHF-02823?func=full-set-set&set_number=000594&set_entry=000001&format=999",
-		"items": [
-			{
-				"itemType": "book",
-				"title": "Registre du dépôt de Nesle: oeuvres d'art saisies pendant la Révolution chez les émigrés et condamnés parisiens en vue de l'instruction publique",
-				"creators": [
-					{
-						"firstName": "Jean-Pierre",
-						"lastName": "Samoyault ",
-						"creatorType": "author"
-					},
-					{
-						"lastName": "Société",
-						"firstName": "de l'histoire de l'art français ",
-						"creatorType": "author"
-					}
-				],
-				"date": "2021",
-				"ISBN": "9782701806174",
-				"language": "fre",
-				"libraryCatalog": "naude.bibliotheque-mazarine.fr Library Catalog",
-				"place": "Paris",
-				"publisher": "éditions de Boccard",
-				"series": "Archives de l'art français",
-				"seriesNumber": "nouvelle période, XLI",
-				"shortTitle": "Registre du dépôt de Nesle",
-				"attachments": [],
-				"tags": [],
-				"notes": [],
-				"seeAlso": []
-			}
-		]
-	},
-	{
-		"type": "web",
-		"url": "https://aleph2.mpg.de/F/FXPBUPI4F7S5QN3M2TSEDBJM5I5INT86VG624HH3BT81E52PYM-05671?func=full-set-set&set_number=004725&set_entry=000002&format=999",
-		"items": [
-			{
-				"itemType": "book",
-				"title": "Zwoelf: die Zeitung der Hochschule für Musik und Theater Hamburg",
-				"creators": [
-					{
-						"lastName": "Hochschule",
-						"firstName": "für Musik und Theater Hamburg",
-						"creatorType": "author"
-					}
-				],
-				"date": "2008",
-				"callNumber": "780",
-				"language": "ger",
-				"libraryCatalog": "aleph2.mpg.de Library Catalog",
-				"place": "Hamburg",
-				"shortTitle": "Zwoelf",
-				"attachments": [],
-				"tags": [
-					{
-						"tag": "Zeitschrift |2 gnd-content"
-					}
-				],
-				"notes": [
-					{
-						"note": "Gesehen am 12.09.17"
-					}
-				],
-				"seeAlso": []
-			}
-		]
-	},
```

Comment:
Are there permalinks for these that we can use instead? Not sure if there's anything different in these worth testing (looks like these are non-English, for what that's worth), but best to assume that tests were added to cover specific situations.

Reply:
Ah, and I guess the French one is from a different library catalog, so would be good to keep that if possible.

Reply:
I added all the tests! It had none before. I’d like to keep them, but I couldn’t find any way to get a permalink. I’m sure I’m just missing something, so I’ll look back at it in a bit.

Reply:
Ah, sorry, was looking at the later commit in isolation.

I can get a permalink if I tweak the URL manually: http://naude.bibliotheque-mazarine.fr/F/?func=direct&doc_number=000319644

I don't know if there's any way for a user to get to that directly, but I guess no harm in adding that, just to have a different installation (assuming that's how Aleph works) tested.

Reply:
Added! Thanks. I wasn't having success making the link myself.